set.mm

```diff
@@ -737500,56 +737500,38 @@ valid cases ( ` (/) ` is the last symbol) and invalid cases ( ` (/) `
   $}
 
   ${
-    $d b x $.  $d b y $.
-    $( Lemma for ~ ichnfimlem3 :  A substitution of a non-free variable has no
-       effect.  Give the distinctor in a form that can be easily eliminiated.
-       Usage of this theorem is discouraged because it depends on ~ ax-13 .
-       (Contributed by Wolf Lammen, 6-Aug-2023.)
-       (New usage is discouraged.) $)
-    ichnfimlem1 $p |- ( ( A. y F/ x ph /\ -. A. x x = y )
-                        -> ( [ a / x ] [ b / y ] ph <-> [ b / y ] ph ) ) $=
-      ( wnf wal weq wn wa wsb wb naev nfsb4t syl5 imp sbft syl ) ABFCGZBCHBGIZJ
-      ACEKZBFZUABDKUALSTUBTBEHBGISUBBCEBMACEBNOPUABDQR $.
-
-    $( Lemma for ~ ichnfimlem3 :  When substituting successively for two always
-       equal variables, the second substitution has no effect.  Usage of this
-       theorem is discouraged because it depends on ~ ax-13 .  (Contributed by
-       Wolf Lammen, 6-Aug-2023.)  (New usage is discouraged.) $)
-    ichnfimlem2 $p |- ( A. x x = y
-                        -> ( [ a / x ] [ b / y ] ph <-> [ b / y ] ph ) ) $=
-      ( weq wal wsb wnf wb nfs1v nfa1 drsb1 nfbidf mpbii sbft syl ) BCFZBGZACEH
-      ZBIZTBDHTJSABEHZBIUAABEKSUBTBRBLABCEMNOTBDPQ $.
-
+    $d b x y $.
     $( Lemma for ~ ichnfim :  A substitution of a non-free variable has no
-       effect.  Usage of this theorem is discouraged because it depends on
-       ~ ax-13 .  (Contributed by Wolf Lammen, 6-Aug-2023.)
-       (New usage is discouraged.) $)
-    ichnfimlem3 $p |- ( A. y F/ x ph
-                    -> ( [ a / x ] [ b / y ] ph <-> [ b / y ] ph ) ) $=
-      ( wnf wal weq wsb wb ichnfimlem2 adantl ichnfimlem1 pm2.61dan ) ABFCGZBCH
-      BGZACEIZBDIQJZPROABCDEKLABCDEMN $.
+       effect.  (Contributed by Wolf Lammen, 6-Aug-2023.)  (Revised by Gino
```

Comment:
Add the reason for the revision: `Avoid ax-13.`, for example

Reply:
Done.